doc/images.md

```diff
@@ -129,3 +129,49 @@ _Note: trimmed sprite sheets are not supported by flame, so if you export your s
 Animations, after created, have an update and render method; the latter renders the current frame, and the former ticks the internal clock to update the frames.
 
 Animations are normally used inside `AnimationComponent`s, but custom components with several Animations can be created as well.
+
+## FlareAnimation
+
+Flame provides a simple wrapper of [Flare](https://www.2dimensions.com/about-flare) animations so you can use them on Flame games.
+
+Check the following snippet on how to use this wrapper:
+
+```dart
+class MyGame extends BaseGame {
+  FlareAnimation flareAnimation;
+  bool loaded = false;
+
+  MyGame() {
+    _start();
+  }
+
+  void _start() async {
+    flareAnimation = await FlareAnimation.load("assets/FLARE_FILE.flr");
```

Comment:
again, could be adding to the component list from base game

Reply:
I have left the `FlameAnimation` and `FlameComponent` example on this game.

Reply:
I like this approach, I'd just add in the text a comment about the fact that normally you should use the Component (exactly like it was done in the Animation section)